apps/arena/lib/arena/game/player.ex

```diff
@@ -315,15 +315,14 @@ defmodule Arena.Game.Player do
     player.aditional_info.inventory != nil
   end
 
-  def use_item(player, game_state, game_config) do
+  def use_item(player, game_state) do
     case player.aditional_info.inventory do
       nil ->
         game_state
 
       item ->
         game_state =
-          Enum.reduce(item.effects, game_state, fn effect_name, game_state_acc ->
-            effect = Enum.find(game_config.effects, fn %{name: name} -> name == effect_name end)
```

Comment:
Why don't we keep this until we do the effects feature?
We could just store the effect name string, so we avoid this transition using the weird and hard-to-read map of effects.

Reply:
You meant the mechanics map right?

Reply:
Yeah